src/hosting/api.ts

```diff
@@ -275,3 +275,82 @@ export async function createRelease(site: string, channel: string, version: stri
   );
   return res.body;
 }
+
+async function getAuthDomains(project: string): Promise<string[]> {
+  const res = await api.request("GET", `/admin/v2/projects/${project}/config`, {
+    auth: true,
+    origin: "https://identitytoolkit.googleapis.com",
+  });
+  return res?.body?.authorizedDomains;
+}
+
+async function updateAuthDomains(project: string, authDomains: string[]): Promise<any> {
```

Comment:
Can a type be defined for these `any`s that are returned? It'll make it much easier for other readers to understand what's being returned.